http-server/src/main/java/io/micronaut/http/server/util/localeresolution/HttpLocaleResolutionConfiguration.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2017-2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.http.server.util.localeresolution;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import io.micronaut.core.annotation.Indexed;
+import io.micronaut.core.util.localeresolution.LocaleResolutionConfiguration;
+
+import java.util.Optional;
+
+/**
+ * Configuration for Locale Resolution in a HTTP Request.
+ *
+ * @author Sergio del Amo
+ * @since 2.3.0
+ */
+@Indexed(HttpLocaleResolutionConfiguration.class)
```

Comment:
This should not be indexed. There will only be 1 bean of this type

Reply:
Is not `@Indexed` necessary because there might be multiple beans of type `LocaleResolutionConfiguration`?

Reply:
The chance of multiple beans here is very low and the there is a cost to indexed so it shouldn't be used in the case of 1 or 2 beans.

Reply:
I've removed it.